CMake/External_VXL.cmake

```diff
@@ -111,6 +95,7 @@ ExternalProject_Add(VXL
     -DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}
     -DCMAKE_C_FLAGS=${CMAKE_C_FLAGS}
     -DCMAKE_INSTALL_LIBDIR:PATH=${fletch_BUILD_INSTALL_PREFIX}/lib
+    -DCMAKE_PREFIX_PATH:PATH=${fletch_BUILD_INSTALL_PREFIX}
```

Comment:
Assuming there are no gotchas that I am not aware of, we might consider adding this line to every project that Fletch is building, or at least the ones that have other dependencies in Fletch.

This is basically telling CMake to prefer finding Fletch packages over system packages.